### ISSUE TITLE
Tabs generation from the song

### ISSUE THREAD

**Issue by nikolayrantsev** [Aturt2/music-sheet-generator#1]
Hi!
Just discovered the library, haven't tried yet, it looks absolutely amazing and fantastic! Thank you for sharing this it, 

Interesting question -  is there any way to upload the song (e.g. "Hotel California") and get the guitar tabs? It seems to be a slicing mechanism like https://github.com/deezer/spleeter working before the tabs generation, but Spleeter not yet trained to extract the guitar tracks,
Appreciate any feedback, thank you!

**Comment by nikolayrantsev** [comment]
Alright, explored a little bit the topic - this is another level of complexity and kind of the task, 
If any ideas - please share:)

**Comment by aomartinezg** [comment]
Hi,

I am glad you found this project interesting!

The tab generation works over a single guitar track, not a song with all of its instruments, so you would have to use a source separation model, like the one you mention, first to isolate the guitar track (not sure how clean the results with these models are, though) or find an individual recording of the guitar. I also have to mention that the project is at an early stage, so, for example, for now it only works with individual notes, it won't detect chords.

Basically, I'm not sure if what you are asking, getting a guitar tab from a full song, is possible right now. However, if you do find a way I would love to know, and if you have any other questions about this project just ask!

Thank you for your kind words!
